toonz/sources/toonzfarm/tfarmcontroller/tfarmcontroller.cpp

```diff
@@ -90,14 +97,16 @@ TFilePath getLocalRoot() {
   TFilePath lroot;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\TOONZROOT");
+  QString regpath = QString::fromStdString("SOFTWARE\\OpenToonz\\OpenToonz\\" + std::string(applicationVersion) + "\\FARMROOT");
```

Comment:
@yamisama, thanks for the feed back and reminding me about the applicationName, I had intended to to that but for got while trying to get it to work.

The original string was prefixed with a 'L' which if I understand correctly is a std::wstring, but in this project it seems QString is preferred. I don't know why the original string was not a standard string and I didn't want to break anything which is why I used QString. It's probably safest to leave it this way unless we can confirm nothing else will be broken.

I'll add in the applicationName change now, any idea how to get the the build server to clone successfully?